src/index.js

```diff
@@ -71,6 +74,18 @@ async function main() {
   const mangataAddress = alice.assets[1].address;
   const turingAddress = alice.assets[2].address;
 
+  const lquidityToken = 'MGR-TUR';
+  console.log(`Checking how much reward available in ${lquidityToken} pool ...`);
+  const rewardAmount = await mangataHelper.calculateRewardsAmount(mangataAddress, lquidityToken);
```

Comment:
this calculateRewardsAmount() call can retrieve the amount remaining in the vault, which is claimable reward.